src/sif/truckcost.cc

```diff
@@ -64,6 +65,28 @@ constexpr float kLeftSideTurnCosts[] = {kTCStraight,         kTCSlight,  kTCUnfa
 // How much to favor truck routes.
 constexpr float kTruckRouteFactor = 0.85f;
 
+constexpr float kMaxHighwayBiasFactor = 8.0f;
```

Comment:
while pulling the entire logic (more or less) into the base class (which I'd prefer) is quite a bit bigger refactor, we can at least declare `kMaxHighwayBiasFactor` in `dynamiccost.cc` so we don't define it 3 times over. actually the arrays are all the same as well, but we can tackle that at a later stage

Reply:
Maybe there would have been possibility for array values becoming different for modes, that they provided them separately in each file. As for Bias Factor I moved it to the header.